test/unit/spec/function_output/kbaseGenericMatrix-spec.js

```diff
@@ -0,0 +1,14 @@
+/*global define*/
+/*global describe, it, expect*/
+/*global jasmine*/
+/*global beforeEach, afterEach*/
+/*jslint white: true*/
+define([
+    'kbaseGenericMatrix'
+], function(Widget) {
+    describe('Test the kbaseGenericMatrix widget', function() {
+        it('Should do things', function() {
+
```

Comment:
Eventually, actual tests should go here. This is better than nothing, for now, since it will check to make sure the widget can be loaded. I'll do a writeup soon on how to do unit tests with our framework.

Reply:
I owe you: more complete tests (after demos)

Reply:
🍻